spec/card_spec.rb

```diff
@@ -19,21 +17,26 @@
           .to(receive(:new))
           .with('cards/getbypan')
           .and_call_original
-        from_pan
```

Comment:
I changed some of the names here to try be more expressive of the action being take, what do you thnk?

Reply:
Yep I agree.  Thanks for this!